Sources/Overload/premake5.lua

```diff
@@ -24,18 +24,27 @@ dependdir = "%{wks.location}/../../Dependencies/"
 builddir = "%{wks.location}/../../Build/"
 resdir = "%{wks.location}/../../Resources/"
 
-include "OvAnalytics"
-include "OvAudio"
-include "OvCore"
-include "OvDebug"
-include "OvMaths"
-include "OvPhysics"
-include "OvRendering"
-include "OvTools"
-include "OvUI"
-include "OvWindowing"
+group "Dependencies"
+    include "../ImGui"
+    include "../tinyxml2"
```

Comment:
Wondering if header only libraries (like sol2), should be added there too? I don't think so, since they cannot technically be compiled. I guess this new "Dependencies" folder should only have sources that can be compiled.